UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneExporter.cs

```diff
@@ -1113,7 +1126,73 @@ private ImageId ExportImage(Texture texture, TextureMapType texturMapType)
 
 			return id;
 		}
+		
+		private ImageId ExportImageInternalBuffer(UnityEngine.Texture texture, TextureMapType texturMapType)
+		{
+
+		    if (texture == null)
+		    {
+			throw new Exception("texture can not be NULL.");
+		    }
 
+		    var image = new GLTFImage();
+		    image.MimeType = "image/png";
+
+		    var byteOffset = _bufferWriter.BaseStream.Position;
+
+		    {//
+			var destRenderTexture = RenderTexture.GetTemporary(texture.width, texture.height, 24, RenderTextureFormat.ARGB32, RenderTextureReadWrite.sRGB);
+			GL.sRGBWrite = true;
```

Comment:
this is only true for basecolor/albedo and emissive.

Reply:
https://github.com/KhronosGroup/UnityGLTF/issues/186  This is the original author，I just made a simple modification. I don’t understand the actual meaning. I don’t know how to change it.